koku/api/metrics/views.py

```diff
@@ -45,14 +51,29 @@ def get_queryset(self):
 
         Restricts the returned data to provider_uuid if supplied as a query parameter.
         """
-        queryset = CostModelMetricsMap.objects.all()
+        try:
+            with open(COST_MODEL_METRICS_FILE_NAME) as json_file:
+                queryset = json.load(json_file)
```

Comment:
so this is no longer queryset.  Let's give it a name that matches what it is now.  QuerySet is a specific django type.

Reply:
updated @kholdaway

Reply:
👍